### ISSUE TITLE
Quests 3.3.4-b316 generated exception when take issued

### ISSUE THREAD

**Issue by smmmadden** [FlyingPikachu/Quests#478]
(3.3.4-b316) What is your Quests version / build number (do _not_ say "latest")?

(issued /quests take stone miner and while in-game it said Quest Accepted, it was cancelled and an exception thrown to console/server) Please describe the problem and how to replicate it. Any console errors?
![image](https://user-images.githubusercontent.com/10309346/45917951-67523480-be4d-11e8-9273-de72aaf55587.png)

![image](https://user-images.githubusercontent.com/10309346/45917936-16424080-be4d-11e8-8975-8790f9c9195f.png)

(default config - no changes) Can you provide an example quest from quests.yml _using Pastebin.com_?
This is the first use of Quests on my 1.13.1 PaperSpigot build 279.  So no changes to configuration files.

1.13.1 issue perhaps?  If the plugin is still using numeric block ID's then that could be the issue.  1.13.x changed/deprecated the use of most block id's in favor of namespaceid's (finally).  Versions listed current versions that support 1.13.x.
My server is a Windows 10 Pro x64 and full list of plugins are:
[09:56:13 INFO]: Plugins (53): Arconix*, AutoBroadcasterPlus*, AutoTorch, BungeeServerSigns*, BungeeTabListPlus*, ColorCodes*, CommandSync*, CorpseReborn*, CreativeGates*, CreeperHeal*, DeluxeJoin, DeluxeMenus*, EditableSign*, EnchantableBlocks, Essentials*, EssentialsAntiBuild*, EssentialsChat*, EssentialsExtras*, EssentialsGeoIP*, EssentialsProtect*, EssentialsSpawn*, HeadDatabase*, HorseTpWithMe, Jobs, Links*, LuckPerms*, LWC, MassiveBooks*, MassiveCore*, Missing_Recipes, OpenInv*, PacketListenerApi*, PlaceholderAPI, ProtocolLib*, PulseHealth*, Quests*, QuickShop, Realistic-Swimming*, Residence, Ridables, SaneEconomy*, Seashells, SecurityVillagers, SelectionVisualizer, Socket4MC*, TintHealth*, TreeAssist, Vault*, Votifier, VotingPlugin, WorldBorder, WorldEdit, WorldSystem

**Comment by smmmadden** [comment]
So I left the game, shutdown the server and started it up again.  This time when I tried to use /quests take stone miner it said I already have that quests.  No other errors in console which was good.  Once I started mining and completed the block breaks, it threw another exception.
![image](https://user-images.githubusercontent.com/10309346/45918222-a4202a80-be51-11e8-9fcc-1f6f42aea426.png)

Also throwing exceptions just using the /quests editor and making changes there.  This sort of confirms it is partially the namespaces are not being used or at least that's the impression. :-)
